docs/99_developer_guide.md

```diff
@@ -2,35 +2,34 @@
 
 You can use a dev container in VSCode to ease development.
 
-### A quickstart video tutorial on how to set up your environment 
+### A quickstart video tutorial on how to set up your environment
 [![Devcontainer quickstart guide](http://img.youtube.com/vi/h05ufOsRgZU/0.jpg)](http://www.youtube.com/watch?v=h05ufOsRgZU "Devcontainer quickstart guide").
 
 
 ### To deploy:
 #### SC64
-* Download the deployer [here](https://github.com/Polprzewodnikowy/SummerCart64/releases/download/v2.20.0/sc64-deployer-windows-v2.20.0.zip)
+* Download the deployer [here](https://github.com/Polprzewodnikowy/SummerCart64/releases/download/v2.20.0/sc64-deployer-windows-v2.20.0.zip).
 * Extract and place `sc64deployer.exe` in the `tools/sc64` directory.
 
-Make sure that your firmware is compatible (currently v2.20.0+)
+Make sure that your firmware is compatible (currently v2.20.0+). 
 See: [here](https://github.com/Polprzewodnikowy/SummerCart64/blob/v2.20.0/docs/00_quick_startup_guide.md#firmware-backupupdate)
 
 ##### From the devcontainer
-It is not currently possible to directly communicate with USB devices.
-BUT, as a workaround you can use a proxy TCP/IP connection
-Set up a proxy: open a terminal window, `cd ./tools/sc64` and then `./sc64deployer.exe server`
+It is not currently possible to directly communicate with USB devices. BUT, you can use a proxy TCP/IP connection as a workaround.  
+To set up a proxy, open a terminal window, `cd ./tools/sc64` and then `./sc64deployer.exe server`
 
-Then in the dev container, use `make run` or `make run-debug`
+Then, in the dev container, use `make run` or `make run-debug`
 
 
 ##### From your host (Windows) OS
 
-* Run `./localdeploy.bat` from the terminal
+* Run `./localdeploy.bat` from the terminal.
 
-Toggle the N64 power switch to load the ROM.
+Toggle the N64 POWER switch to load the ROM.
 
-`ms-vscode.makefile-tools` will help (installed automatically in dev container).
-NOTE: it does not yet work with `F5`: see [this blog post](https://devblogs.microsoft.com/cppblog/now-announcing-makefile-support-in-visual-studio-code/)
-WORKAROUND: in the dev container terminal, use make directly, i.e.: `make`
+`ms-vscode.makefile-tools` will help (installed automatically in dev container).  
+NOTE: it does not yet work with `F5`: see [this blog post](https://devblogs.microsoft.com/cppblog/now-announcing-makefile-support-in-visual-studio-code/).  
+WORKAROUND: in the dev container terminal, use make directly, i.e.: `make`.  
```

Comment:
```suggestion
WORKAROUND: in the dev container terminal, use make directly, i.e.: `make`.
```

Reply:
Double spaces were added to force manual line breaks.
![imagen](https://github.com/user-attachments/assets/7ce7c145-25a3-451e-a8a7-f5adec022c7a)
If you want this to be a single paragraph, then this needs to be rewritten to not follow the current structure.

Reply:
ok, Then it is fine.